lib/helpers.rb

```diff
@@ -41,6 +41,19 @@ module Helpers
       "Reviews" => "isReviewedBy",
     }
 
+    INCLUDED_RELATION_TYPES = [
+      "cites", "is-cited-by",
+      "compiles", "is-compiled-by",
+      "documents", "is-documented-by",
+      "hasmetadata", "is-metadata-for",
```

Comment:
is `hasmetadata` the name of the relation_type_id?

Reply:
fixed in fa01b7bac9294c17c59eb2a9db21f45c4a217d99